packages/angular_devkit/build_angular/src/karma/code-coverage_spec.ts

```diff
@@ -133,7 +146,13 @@ describe('Karma Builder code coverage', () => {
     `);
 
     const run = await architect.scheduleTarget(karmaTargetSpec, { codeCoverage: true });
-    await expectAsync(run.result).toBeResolvedTo(jasmine.objectContaining({ success: false }));
     await run.stop();
+
+    const result = await run.result;
+    // success is true because all tests passed
+    expect(result.success).toBeTrue();
+    const karmaResults: TestResults = result.karmaResults as never;
+    // exitCode is non-zero because coverage failed
+    expect(karmaResults.exitCode).toBe(1);
```

Comment:
With this change, it means that currently with the CLI, there is no way to exit the process with a non zero error code when the